liberty-maven-plugin/src/it/kernel-install-feature-test/prepare-multiple-features-it/pom.xml

```diff
@@ -156,18 +137,12 @@
 
   <dependencyManagement>
     <dependencies>
-      <dependency>
-        <groupId>${bom.group.id}</groupId>
-        <artifactId>hello-bom</artifactId>
-        <version>1.0</version>
-        <type>pom</type>
-      </dependency>
-	<dependency>
-        <groupId>test.user.test.osgi</groupId>
-        <artifactId>SimpleActivator-bom</artifactId>
-        <version>1.0</version>
-        <type>pom</type>
-      </dependency>
+		<dependency>
```

Comment:
Does this mean if they have more than one user feature, they will not be able to configure it because they would both need a `features-bom` artifact in Maven? If so, that seems like an unreasonable restriction.